docs/settings.md

```diff
@@ -72,6 +72,24 @@ Authorization: Bearer ...
 }
 ```
 
+### POST /settings/email/resend
```

Comment:
I think you have removed by mistake the example response of `POST /settings/email`